src/solver/ts-generator/generator.hxx

```diff
@@ -29,12 +29,52 @@
 
 #include <antares/logs/logs.h>
 
-namespace Antares
+namespace Antares::Solver::TSGenerator
 {
-namespace Solver
-{
-namespace TSGenerator
+ 
+inline void ResizeGeneratedTimeSeries(Data::AreaList& areas, Data::Parameters& params)
 {
+    for (auto i = areas.begin(); i != areas.end(); ++i)
```

Comment:
We should use range-based for loop, no need for iterator.

Reply:
Done